### PR TITLE
New: Check live last version from sourceforge.net

### PR DESCRIPTION
**Pull request by ndrosis**

### CODE REVIEW

File: htdocs/admin/tools/update.php [hunk 1]
```diff
@@ -105,7 +105,30 @@
...
+                break;
+            }
+        }
+        $words = array_reverse( $words );
+        $text = implode( " ", $words ) . '';
+    }
+    return $text;
+}
+
+$str = $title;
```
Comment (rdoursenaud) [inline htdocs/admin/tools/update.php:127]: Why do you need to spend another variable?

File: htdocs/admin/tools/update.php [hunk 2]
```diff
@@ -105,7 +105,30 @@
...
+print $langs->trans("LastStableVersion").' : <b>'. word_limiter( $str ).'</b><br>';
+} else {
+    print $langs->trans("LastStableVersion").' : <b>' .('Update Server Off-Line').'</b><br>';
```
Comment (rdoursenaud) [inline htdocs/admin/tools/update.php]: "Update Server Off-Line" should be translatable.
Please use something like

``` php
$langs->trans("UpdateServerOffline")
```

and update the _htdocs/langs/en_US/admin.lang_ file by adding

```
UpdateServerOffline=Update server offline
```

File: htdocs/admin/tools/update.php (hunk 1 above)
Comment (ndrosis) [inline htdocs/admin/tools/update.php:127]: I did not find any other variable in the core to correct showing the version from rss sourceforge.

File: htdocs/admin/tools/update.php (hunk 2 above)
Comment (ndrosis) [inline htdocs/admin/tools/update.php]: Υou're absolutely right, thank you.

**Comment by eldy** [comment]
To load the sourceforge file, can you instead use the function
getURLContent instead of  simplexml_load_file

This will use dolibarr setup for external access (timeout, proxy...).

**Pull request merged**